src/components/cloud-cover/demo/content.twig

```diff
@@ -8,7 +8,7 @@
           level: -2,
           content: 'Our Team'
         } only %}
-        <p>
+        <p class="cloud-cover__description">
```

Comment:
This should be a containing element in order to support multiple classes. We also may need to think about altering the Twig template so this is its own block.

I also am not sure if `description` is too specific. Should it be `__copy`? I'm sorta disappointed that we're already using `content` now... 🤔